src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceCodeFixProvider.cs

```diff
@@ -59,14 +59,16 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
         }
     }
 
-    private IEnumerable<IImplementInterfaceGenerator> GetGenerators(
+    private async IEnumerable<IImplementInterfaceGenerator> GetGenerators(
         Document document, ImplementTypeGenerationOptions options, IImplementInterfaceInfo? state, CancellationToken cancellationToken)
     {
         if (state == null)
```

Comment:
This is a bit weird - can we avoid calling the method if `state` is null?

Reply:
you're looking at old code i think.  it's not like that anymore.